dev/tab1tha/k_nn.py

```diff
@@ -0,0 +1,53 @@
+from sklearn.neighbors import KNeighborsClassifier
+import matplotlib.pyplot as plt
+import numpy as np
+
+def k_nearest(data_train, target_train, data_test, target_test):
+    #training model using the k nearest neighbor algorithm
+    #note that 'neighbor' must be spelt without including a 'u' in order for it to work
+    '''This function trains the model then uses it to predict target values of the test data. In it, the accuracy
+    of the K-nearest neighbor algorithm on this data set is computed and returned '''
+    knn = KNeighborsClassifier(n_neighbors = 5)
+    y = target_train 
+    X = data_train
+
+    #Use data to train the model
+    knn.fit(X,y)
```

Comment:
nit: why cast these variable into shorter, less descriptive, names. You can also pass them directly to knn like so:
`knn.fit(data_train, target_train)`

Reply:
done